app/src/main/java/com/teampophory/pophory/OnBoardingData.kt

```diff
@@ -0,0 +1,5 @@
+package com.teampophory.pophory
+
+data class OnBoardingData (
+    val image : String
+)
```

Comment:
Data 대신 UiModel 정도는 어떨까요?

Reply:
```suggestion
data class OnBoardingData (
    val image : String
) {
    val background: Drawable
        get() = Color.parseColor(image).toDrawable()
}
```
이정도로 사용하면

```kotlin
binding.iv.background = item.background
```

이렇게 사용할 수 있을 것 같네요?